python/tests/test_model_microservice.py

```diff
@@ -240,6 +240,22 @@ def send_feedback_grpc(self, request):
         logging.info("Feedback called")
 
 
+@pytest.fixture(name="mock_get_model_name")
+def fixture_get_model_name(mocker):
```

Comment:
Would it make more sense to mock the environment variables rather than the methods?

Reply:
Having said that, I just realised that you are testing the environment as well down below so maybe you're right in that it makes more sense to mock only the methods here.